orchid.gemspec

```diff
@@ -10,8 +10,8 @@ Gem::Specification.new do |s|
   s.authors     = ["Jessica Dussault", "Greg Tunink", "Karin Dalziel"]
   s.email       = ["jdussault4@gmail.com", "techgique@gmail.com", "kdalziel@unl.edu"]
   s.homepage    = "https://cdrh.unl.edu"
-  s.summary     = "Template for Pando sites"
-  s.description = "Template for sites powered off of Pando API"
+  s.summary     = "Template for CDRH API sites"
```

Comment:
I'm guessing this was happening about the same time we were looking around my office saying random words, so I think it's okay to remove Pando, haha